### ISSUE TITLE
Update Collections Privacy Switch button to look like a button

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#10128]
### Is your proposal related to a problem?

The 'status' tag element is used throughout the Wagtail admin to indicate the status of an object (usually some kind of active/disabled/live/draft type status).

The privacy indicator on the admin page editor no longer uses the status styling to indicate whether it's public/private, but the collection edit page still has the original status-like designs.

<img width="1988" alt="Screenshot 2023-02-21 at 10 31 40 pm" src="https://user-images.githubusercontent.com/1396140/220345765-5eb08058-c791-428e-a59b-d97a177d3151.png">

This creates a disconnect in the presentation of what status elements are and does not align with other small buttons used throughout the admin.

### Describe the solution you'd like

Update this file `wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html` to avoid using `status-tag` but instead use small button styling to infer that it's a button.

`button button-small button-secondary` (with icon) should be sufficient to replace the status-tag presentation here.

Here is a quick inline edit in the DOM.

<img width="1988" alt="Screenshot 2023-02-21 at 10 31 40 pm" src="https://user-images.githubusercontent.com/1396140/220346610-57d1fdda-b458-45f2-b222-9dd11ef6b0f6.png">
            

### Describe alternatives you've considered

* We can try to make the status tag work for 'button' also but this seems odd for one left over edge case.

### Additional context

* _Originally posted by @lb- in https://github.com/wagtail/wagtail/issues/9721#issuecomment-1438356041_
* Follow on from #8656

**Comment by gasman** [comment]
Thanks for your interest @JatinKumar001! Yes, you are welcome to work on this. We do not assign issues outside of the core team - we just ask that people indicate their intention to work on an issue (as you have now done), to avoid duplicating efforts.

For further help in getting started, please see https://dev.to/lb/ten-tasty-ingredients-for-a-delicious-pull-request-hgc - you may also wish to join [our #new-contributors channel on Slack](https://github.com/wagtail/wagtail/wiki/Slack).

**Comment by Pratik-A-N** [comment]
Hello, can I work on this issue ??

**Comment by lb-** [comment]
@Pratik-A-N please take the time to read through our WIP guide for your first contribution. It talks about how to ensure that an issue is ready and to make sure you read through things like PRs that are already up / already being worked on. 

https://github.com/wagtail/wagtail/issues/10070

**Comment by Pratik-A-N** [comment]
Sure

**Comment by JatinKumar001** [comment]
@Pratik-A-N I already resolve this issue.
